File: 912-sort-an-array/912-sort-an-array.cpp
class Solution {
public:
    vector<int> sortArray(vector<int>& nums) {
             priority_queue<int,vector<int>,greater<int>>pq;
        for(auto it:nums){
            pq.push(it);
        }
       vector<int> arr;
        
        while(pq.size()>0){
            arr.push_back(pq.top());
            pq.pop();
           
            
        }
        return arr;
    }
};

File: 912-sort-an-array/912-sort-an-array.java
class Solution {
    public int[] sortArray(int[] nums) {
       PriorityQueue<Integer>pq=new PriorityQueue<>();
        for(int it:nums){
            pq.add(it);
        }
        int[] arr=new int[nums.length];
        int i=0;
        while(pq.size()>0){
            arr[i]=pq.poll();
            i++;
            
        }
        return arr;
    }
}